flow/activities/flowable_core.go

```diff
@@ -186,7 +186,7 @@ func syncCore[TPull connectors.CDCPullConnectorCore, TSync connectors.CDCSyncCon
 	var res *model.SyncResponse
 	errGroup.Go(func() error {
 		syncBatchID, err := dstConn.GetLastSyncBatchID(errCtx, flowName)
-		if err != nil && config.Destination.Type != protos.DBType_EVENTHUBS {
```

Comment:
@Amogh-Bharadwaj do you know why this was here?

Reply:
I think we can remove this @serprex